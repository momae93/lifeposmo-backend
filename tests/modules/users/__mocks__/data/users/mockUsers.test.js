const mockData = require('../../../../../../src/modules/users/__mocks__/data/users/mockUsers');

describe('[MODULES][USERS][__MOCKS__][DATAS] : mock Users', () => {
  it('should match snapshot mock users', async () => {
    const mockUsers = mockData;

    expect(mockUsers).toMatchSnapshot();
  });
});
