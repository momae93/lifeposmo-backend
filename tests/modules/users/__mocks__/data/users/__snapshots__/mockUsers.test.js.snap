// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[MODULES][USERS][__MOCKS__][DATAS] : mock Users should match snapshot mock users 1`] = `[Function]`;
